### PR TITLE
Doc'd support for assistive technologies in the admin.

### PR DESCRIPTION
**Pull request by thibaudcolas**
See also #17338. I thought it was important to separately acknowledge what the goals and current state are for the Django admin, similarly to the FAQ entry about browser support. 

Compared to contributing docs, this is for a very different audience of Django users so I chose to have this completely separate (and much more digestible), though the information is similar.

---

Like #17338, this is heavily based on:

- [Wagtail’s Assistive technologies support](https://guide.wagtail.org/en-latest/reference/browser-compatibility/#assistive-technologies)
- The UK Government Digital Services’ guidance on [Testing with assistive technologies](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies), itself based on one of the most comprehensive public [survey of assistive technology users](https://accessibility.blog.gov.uk/2016/11/01/results-of-the-2016-gov-uk-assistive-technology-survey/) I’m aware of.

### CODE REVIEW

File: docs/faq/admin.txt [hunk 1]
```diff
@@ -109,3 +109,12 @@ Android and iOS.
...
 between browsers. These are considered acceptable variations in rendering.
+
+What assistive technologies are supported for using the admin?
+==============================================================
+
+We want the admin to be compatible with wide range of assistive
+technologies, but are aware of many blockers currently. We intend to support
+all latest versions of major assistive technologies, including
+JAWS, NVDA, VoiceOver iOS, TalkBack, Dragon, VoiceOver macOS, Voice Control,
+ZoomText, Windows Contrast Themes, and screen magnifiers.
```
Comment (felixxm) [inline docs/faq/admin.txt:120]: Can we somehow avoid "we"?

Comment (felixxm) [inline docs/faq/admin.txt:120]: Do you think it's worth mentioning so many utilities? :thinking:

Comment (felixxm) [reply docs/faq/admin.txt:120]: For example: `The admin is intended to be compatible with ...`

Comment (thibaudcolas) [reply docs/faq/admin.txt:120]: Not necessarily, I just did this to match what’s done right above for browser support. I’d have felt odd to me to single out specific browsers but not assistive tech.

Comment (thibaudcolas) [reply docs/faq/admin.txt:120]: 👍 I can try if we want to, but was also trying hard to avoid passive voice :) I’ll try to think of another sentence.

Comment (felixxm) [reply docs/faq/admin.txt:120]: Fine for me, but you have to promise to remove unmaintained options from this list in the future :wink:

Comment (thibaudcolas) [reply docs/faq/admin.txt:120]: I promise!

Comment (shangxiao) [inline docs/faq/admin.txt:120]:
```suggestion
ZoomText, Windows Contrast Themes and screen magnifiers.
```

Just FYI commas before "and" is only correct when joining independent clauses.

File: docs/faq/admin.txt [hunk 2]
```diff
@@ -109,3 +109,12 @@ Android and iOS.
 
 Depending on feature support, there *may* be minor stylistic differences
 between browsers. These are considered acceptable variations in rendering.
+
+What assistive technologies are supported for using the admin?
+==============================================================
+
+We want the admin to be compatible with wide range of assistive
+technologies, but are aware of many blockers currently. We intend to support
```
Comment (shangxiao) [inline docs/faq/admin.txt:117]:
```suggestion
technologies, but are aware of many current blockers. We intend to support
```
What's the intended meaning of "are aware of many blockers currently"? 🤔 The blockers are current? If so then "many current blockers".

File: docs/faq/admin.txt (hunk 1 above)
Comment (thibaudcolas) [reply docs/faq/admin.txt:120]: Doesn’t the removal of the comma make it sound like screen magnifiers are only a Windows thing?

File: docs/faq/admin.txt (hunk 2 above)
Comment (thibaudcolas) [reply docs/faq/admin.txt:117]: 👍 "we are currently aware of many blockers" sounds much better

File: docs/faq/admin.txt (hunk 1 above)
Comment (shangxiao) [reply docs/faq/admin.txt:120]: Actually you're originally right, looks like _not_ using the "serial comma" is a thing in my country (and I was wrong about it "not being correct" as it's still acceptable) but it's common in US english. 😅

It's better to use it, as they say in The Zen of Python: "Explicit is better than implicit." 👍

File: docs/faq/admin.txt [hunk 3]
```diff
@@ -109,3 +109,12 @@ Android and iOS.
 
 Depending on feature support, there *may* be minor stylistic differences
 between browsers. These are considered acceptable variations in rendering.
+
+What assistive technologies are supported for using the admin?
```
Comment (knyghty) [inline docs/faq/admin.txt:113]: I'm nor sure this reads well, maybe:

```suggestion
What assistive technologies are supported in the admin?
```

File: docs/faq/admin.txt (hunk 1 above)
Comment (knyghty) [reply docs/faq/admin.txt:120]: I wonder if it's maybe better to mention classes of technologies, "screen readers" rather than listing JAWS, NVDA, etc.

Comment (thibaudcolas) [reply docs/faq/admin.txt:120]: @knyghty could you explain why? From my perspective it felt important to match what is done right above for browser support, where I imagine we could also just say "modern browsers" for simplicity but it’s probably clearer for everyone if we mention specific names.

Comment (knyghty) [reply docs/faq/admin.txt:120]: Well, fine for me, but you have to promise to remove unmaintained options from this list in the future... and add new ones. Do you think you'll remember?

File: docs/faq/admin.txt (hunk 3 above)
Comment (Scotchester) [reply docs/faq/admin.txt:113]: Perhaps "by the admin"?

File: docs/faq/admin.txt [hunk 4]
```diff
@@ -109,3 +109,12 @@ Android and iOS.
...
+==============================================================
+
+We want the admin to be compatible with wide range of assistive
```
Comment (Scotchester) [inline docs/faq/admin.txt:116]:
```suggestion
We want the admin to be compatible with a wide range of assistive
```

File: docs/faq/admin.txt (hunk 1 above)
Comment (thibaudcolas) [reply docs/faq/admin.txt:120]: Promise

**Comment by thibaudcolas** [comment]
This is ready for another round of review :)

**Review by felixxm — APPROVED** [review]
@thibaudcolas Thanks :+1: 

I changed `VoiceOver iOS` and `VoiceOver macOS` to single `VoiceOver`, and alphabetized the list.

**Comment by thibaudcolas** [comment]
Thank you @felixxm! I believe it’s important we have VoiceOver listed separately for iOS and macOS though. I have opened #17485 to address this.

**Pull request merged**